kitsune/wiki/widgets.py

```diff
@@ -1,11 +1,17 @@
-import collections
+import sys
+
 
 from django import forms
 from django.template.loader import render_to_string
 
 from kitsune.products.models import Topic
 from kitsune.wiki.models import Document
 
+if sys.version_info >= (3,3):
```

Comment:
Could you replace the `if/else` statement with:

```python
try:
    from collections.abc import Iterable
except ImportError:
    from collections import Iterable
``` 

just to be consistent with our imports?

Reply:
Will do!

Reply:
Updated :)